packages/@ember/object/mixin.ts

```diff
@@ -8,12 +8,14 @@ import { guidFor, observerListenerMetaFor, ROOT, wrap } from '@ember/-internals/
 import { assert } from '@ember/debug';
 import { DEBUG } from '@glimmer/env';
 import { _WeakSet } from '@glimmer/util';
-import type {
-  ComputedDecorator,
-  ComputedPropertyGetter,
-  ComputedPropertyObj,
-  ComputedPropertySetter,
-  ComputedDescriptor,
+import {
+  type ComputedDecorator,
+  type ComputedPropertyGetter,
+  type ComputedPropertyObj,
+  type ComputedPropertySetter,
+  type ComputedDescriptor,
+  isAccessor,
```

Comment:
You imported this but didn't use it *and* it doesn't appear to be used below, so I think we can simply drop this line from the `import` statement?

```suggestion
```

Reply:
Interestingly, the [files tab](https://github.com/emberjs/ember.js/pull/20388/files) of this PR is showing linter (tsc) errors related to this, but they didn't fail CI. Is that configuration intentional?